src/core_ocean/Registry.xml

```diff
@@ -693,6 +693,31 @@
 					description="The length scale of exponential decay of river runoff. Fluxes are multiplied by $e^{z/\gamma}$, where this coefficient is $\gamma$."
 					possible_values="Any positive real number."
 		/>
+                <nml_option name="config_use_forcing" type="logical" default_value=".true." units="unitless"
```

Comment:
This option could presumably be used to activate a package but I don't think that's being done at the moment, correct?

Reply:
Also, do we need individual config options for using atmospheric forcing, sea ice forcing, etc?  I could imagine we might want some but not others.  Maybe, then, it makes sense to have a package for each type of forcing rather than one package for all time-variable forcing.

Reply:
@xylar, I think having different packages for different time-variable forcing makes sense.